.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
```

Comment:
Would it make sense to call this file CI.yaml to match the name?

Reply:
Maybe. FWIW this is the default template Github generates for "set up a workflow yourself":

![image](https://user-images.githubusercontent.com/557590/84923889-bb8c5600-b0bf-11ea-822f-6afc2f3b8581.png)

So I assumed it's a convention for default filename / name?